c2corg_ui/format/ltag.py

```diff
@@ -16,9 +16,14 @@
 class LTagPreprocessor(Preprocessor):
     """ Preprocess lines to clean LTag blocks """
 
+    RE_LTAG_UNSUPPORTED = re.compile(r'[L|R]#[^\s|]+')
```

Comment:
Could you add a comment to precise what L# syntaxes are actually supported (and what are not)?